src/AParser.hs

```diff
@@ -61,7 +63,7 @@ posInt = Parser f
 -- Exercise 1: Functor instance for Parser
 -----------------------------------------------------------------
 first :: (a -> b) -> (a,c) -> (b,c)
-first f (arg1, arg3) = (f arg1, arg3)
+first f (arg1, arg3) = BF.first f (arg1, arg3)
 
 instance Functor Parser where
   -- fmap :: (a -> b) -> Parser a -> Parser b
```

Comment:
You can use use `BF.first` in `fmap (first f) . a`. Or write `first = BF.first` but that's not important.